doc/installing-from-release-windows.md

```diff
@@ -11,161 +11,39 @@
 
 # `install.ps1`
```

Comment:
instead of duplicating this in the doc, maybe let's just link to the script?

Reply:
agreed